austation/code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -260,6 +260,10 @@
 			return FALSE
 	return ..()
 
+/obj/item/robot_module/butler/Initialize()
+	..()
+	basic_modules |= /obj/item/storage/bag/tray
```

Comment:
oh god yes
look at this modularization
![image](https://user-images.githubusercontent.com/11001588/108929923-091e8100-7699-11eb-9152-433b7f01fd08.png)

Reply:
I don't get it, sorry.

Reply:
good modularization makes me wanna coooooom

Reply:
Reading this again, should I instead make it +=, rather than |= ?

Reply:
Or maybe keep the |= operator and make it a list containing 1 item?

Reply:
|= is the correct way to do this